### PR TITLE
Core/readme file

### PR DESCRIPTION
**Pull request by LimbertVargas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,25 @@
-# trello-python-tests
+# Trello API Project
+This project allows testing of the Trello application using Rest API.
```
Comment (Enrique-C) [inline README.md]: It would be ".....allows to test Trello application using Behave"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,25 @@
...
+
+##Installing
+To start setting up the environment with BDD (Behavior Driven Development).
+* Install [Python 3.8.1](https://www.python.org/downloads/)
+* Install [PyCharm 2019.3.2 (Professional Edition)](https://www.jetbrains.com/es-es/pycharm/download/#section=windows) 
+* Install [Git version 2.21.0.windows.1](https://git-scm.com/download/win)
+
+## Configuration
+List of required libraries in Project interpreter
+* Install [behave](https://behave.readthedocs.io/en/stable/) interpreter in PyCharm.
```
Comment (Enrique-C) [inline README.md]: We don't use this URL, we add the library from Pycharm

File: README.md [hunk 3]
```diff
@@ -14,12 +14,10 @@ To start setting up the environment with BDD (Behavior Driven Development).
 
 ## Configuration
 List of required libraries in Project interpreter
-* Install [behave](https://behave.readthedocs.io/en/stable/) interpreter in PyCharm.
+* Install [behave](https://www.jetbrains.com/help/pycharm/creating-feature-files.html) interpreter in PyCharm.
 * Install [requests](https://pypi.org/project/requests/)
 * Install [jsonschema](https://pypi.org/project/jsonschema/)
 * Install [Pytest](https://docs.pytest.org/en/latest/getting-started.html) to write unit tests
```
Comment (Amet015) [inline README.md]: all of this we install through pycharm

Comment (LimbertVargas) [reply README.md]: yes because is the configuration of the IDE

Comment (Enrique-C) [reply README.md]: But it would show the procedure in the IDE, not like a link that needs to be downloaded

File: README.md [hunk 4]
```diff
@@ -13,11 +13,24 @@ To start setting up the environment with BDD (Behavior Driven Development).
 * Install [Git version 2.21.0.windows.1](https://git-scm.com/download/win)
 
 ## Configuration
-List of required libraries in Project interpreter
-* Install [behave](https://www.jetbrains.com/help/pycharm/creating-feature-files.html) interpreter in PyCharm.
-* Install [requests](https://pypi.org/project/requests/)
-* Install [jsonschema](https://pypi.org/project/jsonschema/)
-* Install [Pytest](https://docs.pytest.org/en/latest/getting-started.html) to write unit tests
+This project was intended to work with python 3.8.1 A list of required libraries can be found on "requirements.txt" 
+file.
```
Comment (Enrique-C) [inline README.md:17]: Would it be in code line 16?

**Review by Enrique-C — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1,2 +1,36 @@
...
 
+##Prerequisites
+For the correct functioning of the work environment, the following programs need to be installed:
+* Windows Environment: Server 2019 Standard.
+* Google Chrome version 79.0.3945.130
+
+##Installing
+To start setting up the environment with BDD (Behavior Driven Development).
+* Install [Python 3.8.1](https://www.python.org/downloads/)
+* Install [PyCharm 2019.3.2 (Professional Edition)](https://www.jetbrains.com/es-es/pycharm/download/#section=windows)
```
Comment (carledriss) [inline README.md]: remove this

File: README.md [hunk 6]
```diff
@@ -1,2 +1,36 @@
...
+##Prerequisites
+For the correct functioning of the work environment, the following programs need to be installed:
+* Windows Environment: Server 2019 Standard.
+* Google Chrome version 79.0.3945.130
+
+##Installing
+To start setting up the environment with BDD (Behavior Driven Development).
+* Install [Python 3.8.1](https://www.python.org/downloads/)
+* Install [PyCharm 2019.3.2 (Professional Edition)](https://www.jetbrains.com/es-es/pycharm/download/#section=windows) 
+* Install [Git version 2.21.0.windows.1](https://git-scm.com/download/win)
```
Comment (carledriss) [inline README.md]: remove this

File: README.md [hunk 7]
```diff
@@ -1,2 +1,36 @@
...
+This means all required libraries can be installed from the project path using:
+```bash 
+pip install -r requirements.txt
+```
+## About the Project
+On the 'core' folder we intend to put generic steps and functions that can be used on any API testing. 
+We also included a generic API request class, this class can be inherited to customize request options.
+## Execution
+To execute this project we are focus on the features, to run any scenario we proceed to select the feature directory 
+and select any feature, right click in the feature file and click in Run.
```
Comment (carledriss) [inline README.md:45]: Please add the command line

File: README.md [hunk 8]
```diff
@@ -1,2 +1,36 @@
...
+To finish this configuration we need add Pytest in Python Integrated Tool follow the next steps:
+
+Go File -> Settings -> Tools -> Python Integrated Tools and in Testing field in Default test runner select pytest option.  
+
+This means all required libraries can be installed from the project path using:
+```bash 
+pip install -r requirements.txt
+```
+## About the Project
+On the 'core' folder we intend to put generic steps and functions that can be used on any API testing.
```
Comment (carledriss) [inline README.md:32]: Could you mention how to configure config.json

**Review by carledriss — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (LimbertVargas) [reply README.md:17]: yes, but I put it down so as not to pass the delimitation line.

File: README.md (hunk 5 above)
Comment (LimbertVargas) [reply README.md]: done

File: README.md (hunk 6 above)
Comment (LimbertVargas) [reply README.md]: done

File: README.md (hunk 8 above)
Comment (LimbertVargas) [reply README.md:32]: done

File: README.md (hunk 7 above)
Comment (LimbertVargas) [reply README.md:45]: done

File: README.md [hunk 9]
```diff
@@ -20,17 +20,40 @@ file.
...
+
+```
+token:
+```
+This field is fill with the key provided in Trello API documentation
+
+```
+email:
+``` 
+This field is fill with a valid email used to loggin in Trello application
```
Comment (Enrique-C) [inline README.md]: It is a mail to test features only, not to login in trello

**Review by Enrique-C — APPROVED** [review]

File: README.md (hunk 9 above)
Comment (LimbertVargas) [reply README.md]: done

**Review by Amet015 — APPROVED** [review]

**Review by carledriss — APPROVED** [review]

**Pull request merged**